Lib/pydoc.py

```diff
@@ -1892,9 +1894,9 @@ def __repr__(self):
                                      self.__class__.__qualname__)
 
     _GoInteractive = object()
-    def __call__(self, request=_GoInteractive):
+    def __call__(self, request=_GoInteractive, follow_wrapped=False):
```

Comment:
Can you explain why this should be `False` by default?

Reply:
Yes, thinking better, here must be a `follow_wrapped=True`